test/lib/mayaUsd/nodes/testProxyShapeBase.py

```diff
@@ -113,6 +115,8 @@ def testDuplicateProxyStageAnonymous(self):
         self.assertEqual(0, len(ufe.Hierarchy.hierarchy(duplProxyShapeItem).children()))
         self.assertEqual(1, len(ufe.Hierarchy.hierarchy(proxyShapeItem).children()))
 
+    # protect against Maya flavor: Maya PR 121 now has support for UFE path string.
+    @unittest.skipUnless(((ufeUtils.ufeFeatureSetVersion() >= 2) and (mayaUtils.previewReleaseVersion() >= 121)), 'testDuplicateProxyStageFileBacked is only available in Maya Preview Release 121 or later.')    
     def testDuplicateProxyStageFileBacked(self):
```

Comment:
- moved "ufeSelectCmd" under "testUtils" so it can be used in "test/lib/mayaUsd"
- added protection against Maya flavor: Maya PR 121 now has support for UFE path string.

Reply:
Path strings have been in UFE v2 for a long time...  Why isn't ufeFeatureSetVersion() >= 2 sufficient?

Reply:
I was following what we had in testSelection. Should we clean it up too?

https://github.com/Autodesk/maya-usd/blob/80cddd12c85cf0678b758484e98d5ffb0ea58f4e/test/lib/ufe/testSelection.py#L140

Reply:
I think in that case it was to protect for availability of the Maya select command improvements to support selection with path strings.  Since we're now at preview release 122 (soon 123), we should go back and clean all those obsolete preview release conditional test executions, in a cleanup pull request, no doubt.